app/services/void_participant_declaration.rb

```diff
@@ -1,34 +1,46 @@
 # frozen_string_literal: true
 
 class VoidParticipantDeclaration
-  attr_accessor :cpd_lead_provider, :id
+  attr_reader :participant_declaration
 
-  def initialize(cpd_lead_provider:, id:)
-    @cpd_lead_provider = cpd_lead_provider
-    @id = id
+  def initialize(participant_declaration:)
+    @participant_declaration = participant_declaration
   end
 
   def call
+    if participant_declaration.paid?
+      make_awaiting_clawback
+    else
+      make_voided
+    end
+
+    Api::V1::ParticipantDeclarationSerializer.new(participant_declaration).serializable_hash.to_json
```

Comment:
while we're here any chance we can make this return `participant_declaration` instead and let the controller decide what serialiser to use?